src/worldofzuul/Parser.java

```diff
@@ -15,7 +16,10 @@ public Parser() {
         commands = new CommandWords();
         reader = new Scanner(System.in);
     }
-
+    /**
+    * @return
+    * It reads word1 and makes a new Scanner to make a inputLine
```

Comment:
```java
Din docblock skal sættes op som herunder.
/**
 * [beskrivelse]
 * @return [Hvad retur værdi har methoden] 
 */

Reply:
Fix in: 3817033c374b549f137013ed3b42d7a4517cfe25